api/src/env.ts

```diff
@@ -7,7 +7,7 @@ import dotenv from 'dotenv';
 import fs from 'fs';
 import { clone, toNumber, toString } from 'lodash';
 import path from 'path';
-import logger from './logger';
+import logger from './logger-no-env';
```

Comment:
Lets figure out a way we can resolve the circular problem in a way that doesn't require a duplicate of the same logger 🤔 

I think we can resolve it by requiring it later in the flow with an inline require